README.rst

```diff
@@ -98,6 +98,8 @@ information.
    :target: https://travis-ci.org/coala-analyzer/coala-bears
 .. |Documentation Status| image:: https://readthedocs.org/projects/coala/badge/?version=latest
    :target: http://coala.rtfd.org/
+.. |codecov.io| image:: https://img.shields.io/codecov/c/github/coala-analyzer/coala/master.svg?label=branch%20coverage
+   :target: https://codecov.io/github/coala-analyzer/coala-bears
```

Comment:
though `https://img.shields.io/codecov/c/github/coala-analyzer/coala/master.svg` should definitely be changed to coala-bears instead of coala